Public/Objects/Compare-ObjectProperties.ps1

```diff
@@ -4,16 +4,18 @@ Function Compare-ObjectProperties {
         [PSObject]$DifferenceObject,
         [switch]$CaseSensitive = $false
```

Comment:
switch by default is $false.

Reply:
yes, because if I leave it empty, it is caseSensitive by default

Reply:
can we leave it $false?

Reply:
Ok, ye because of $CaseSensitive.IsPresent is actually the value you are checking against, and it's not present if a person doesn't use it.

Reply:
We can, I wasn't even aware you're able to do that!